.github/workflows/validate-pr.yml

```diff
@@ -35,27 +35,24 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-yarn-
 
-      - name: Cache Gatsby cache folder
+      - name: Cache Gatsby build
         uses: actions/cache@v2
         with:
-          path: .cache
-          key: ${{ runner.os }}-cache-gatsby-${{ github.sha }}
+          path: |
+            public
```

Comment:
I assume this comes out to be something like this: `public/.cache`

Reply:
Not quite. By putting these on different lines, it's caching multiple paths. You can check it out on [the documentation](https://github.com/actions/cache#whats-new)